nie-ine/nie-frontend
# Nie 334 wysiwyg angular

To test:
 - click on login
 - click on brief edieren task
 - you see a white desktop, click on nie - ine apps
 - a sidenav opens, click on image viewer
 - a button opens with the name image viewer and an index, click on this button
 - a window with the image viewer opens, you can use Retos image viewer
 - You can close the window, the app with the index stays on the right side
 - you can click on the settings and the app instance on the right closes as well
 - you can add an arbitrary amount of the same app, if you close one app the other apps stay open.
